allowance_unece/models/unece_code_list.py

```diff
@@ -0,0 +1,13 @@
+# © 2020 Decodio Applications ltd (https://decod.io)
+# @author: Davor Bojkic <davor.bojkic@decod.io>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models, fields
+
+
+class UneceCodeList(models.Model):
+    _inherit = 'unece.code.list'
+
+    type = fields.Selection(selection_add=[
```

Comment:
Every time you make a `selection_add`, you should provide an `uninstall_hook` to provide clean uninstall. See for example https://github.com/odoo/odoo/issues/45240#issuecomment-585727789. You can find many examples of uninstall hook in OCA modules.

Reply:
i agree that selection-add can be tricky, but none of modules extending unece_code_list model have such hook.. 
it adds a data sorted according to field selection.. if module is uninstalled, all functional domains on model unece_code_list used by module are not functional, so even if data is left after uninstall process it will not imact on anything... 
check in modules : account_tax_unece, account_payment_unece
still thinking we need uninstall hook?

Reply:
You are right. This selection is not functionally used, so it doesn't cause any error.